src/mechanics/mesh/MeshFem.h

```diff
@@ -11,13 +13,63 @@ namespace NuTo
 class MeshFem
 {
 public:
-
     InterpolationSimple& CreateInterpolation(const InterpolationSimple& interpolation)
     {
         mInterpolations.push_back(interpolation.Clone().release());
         return *mInterpolations.rbegin();
     }
 
+    //! @brief selects a node of type \p dofType at given \p coords
```

Comment:
Doxygen understands Markdown. I'd prefer

    selects a node of type `dofType` at given `coords`

Reply:
Good point. I actually thought that `\p` is somehow related to `@param` and somehow creates a hyperref from the word to the parameter. If it is just about the font, we could leave it out completely. What do you think?

Reply:
Unfortunately, it's just the font. In the generated HTML, the difference is almost unnoticeable. But I'm so used to this syntax that I find it useful in the source to differentiate between "prose" and "code".